src/analysis/modals/ExportAnalysisModal/ExportAnalysisModal.test.ts

```diff
@@ -155,15 +160,13 @@ describe('ExportAnalysisModal', () => {
     );
 
     // Act
-    const destDropdown = screen.getByLabelText('Destination *');
-    await user.click(destDropdown);
```

Comment:
Why the removal of the user.click action? Was it redundant?

Reply:
This was clicking on the dropdown to show the options menu, reading the text of each option, then clicking the "name2" option to select it.

I replaced these lines with SelectHelper, which provides a higher level interface (`getOptions` and `selectOption`) to do the same thing. I'll claim that this is more readable.